src/test/run-pass/rfc-1014.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+#![feature(libc)]
+
+extern crate libc;
+
+#[cfg(windows)]
+fn close_stdout() {
+    pub const STD_OUTPUT_HANDLE: libc::DWORD = -11i32 as libc::DWORD;
+    unsafe { libc::CloseHandle(STD_OUTPUT_HANDLE); }
```

Comment:
1.  `STD_OUTPUT_HANDLE` isn't a `HANDLE` so you need to `GetStdHandle` to get the actual `HANDLE` to `CloseHandle`.

2. Write a separate test which does `SetStdHandle(STD_OUTPUT_HANDLE, NULL)` so it can test `println!`'s ability to replace stdout with a sink, rather than just stdout's ability to ignore errors.